dev/Deprecation.php

```diff
@@ -116,6 +123,34 @@ protected static function get_called_method_from_trace($backtrace, $level = 1) {
 			return $called['function'];
 		}
 	}
+	
+	/**
+	 * Determine if deprecation notices should be displayed
+	 * 
+	 * @return boolean
+	 */
+	public static function get_enabled() {
+		// Never raise deprecation notices in a live environment
+		if(Director::isLive(true)) {
```

Comment:
@dhensby agreed, let's make it !isDev(). Test sites shouldn't show deprecation notices.